taichi/codegen/llvm/codegen_llvm.h

```diff
@@ -44,6 +44,11 @@ class FunctionCreationGuard {
   ~FunctionCreationGuard();
 };
 
+struct LLVMCompiledData {
```

Comment:
We probably wanna merge this class with "OffloadedTaskCacheData" (in some future PRs maybe)?

Reply:
Just noticed the following PR. plz ignore this one.